test/e2e/specs/editor/various/popovers.spec.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'new editor state', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.activatePlugin(
+			'gutenberg-test-plugin-post-formats-support'
+		);
```

Comment:
I just found out that this isn't included in the original test, I think we should remove it (along with `test.beforeAll` and `test.afterAll`). Seems like just a copy-paste error :)